portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -195,12 +195,57 @@ definition {
 	}
 
 	@description = "Verify that inputs to the Body Text can be undone and redone"
-	@ignore = "true"
 	@priority = "5"
 	test CanUndoAndRedo {
 		property portal.acceptance = "false";
```

Comment:
Can you please add `property forms.functional = "true";` here following alphabetic order?

I mean, something like:
```
property forms.functional = "true";
property portal.acceptance = "false";
```

Thanks.